### ISSUE TITLE
Maximum / minimum number of specific blocks

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Checklists#148]
This a requirement that could be used multiple times in one checklist:

The rule would look like this: "Add minimum/maximum number of blocks of X type"

Examples:

- Add a minimum of 1 heading block
- Add a maximum of 3 image blocks
- Add a minimum of 5 paragraph blocks

**Comment by andergmartins** [comment]
Maybe we can use an approach similar to what we did for custom taxonomies. We would detect the registered blocks and show one option for each. 
The tasks list can become long.

**Comment by stevejburge** [comment]
Yes, very long. That might require the new interface https://github.com/publishpress/PublishPress-Checklists/issues/176

**Comment by stevejburge** [comment]
Two more requests for this, but neither is using Gutenberg. We'd have to do this at the level of HTML tags

**Comment by stevejburge** [comment]
More:

> there will be lot of users on my site who will be author uploading their posts. i want to restrict them to upload maximum 5 images / videos per post.

**Comment by stevejburge** [comment]
More:

> I want to know if we can force Contributor/author etc to add 2-3 images in the post body.
> We have quite a few blogs, but people dont add images , we want a way to enforce 2-4 images in the content.
